apps/prairielearn/src/pages/partials/attachFilePanel.ejs

```diff
@@ -11,7 +11,7 @@
       <% file_list.forEach(function(file, iFile) { %>
       <tr>
         <td style="word-break:break-all;">
-          <a class="attached-file" href="<%= urlPrefix %>/assessment_instance/<%= assessment_instance.id %>/file/<%= file.id %>/<%= file.display_filename %>">
+          <a class="attached-file" href="<%= plainUrlPrefix %>/course_instance/<%= course_instance.id %>/assessment_instance/<%= assessment_instance.id %>/file/<%= file.id %>/<%= file.display_filename %>">
```

Comment:
This was needed because `urlPrefix` in the manual grading context includes `/instructor`. An alternative is to create a separate endpoint and use that here if the context is manual grading, but an instructor that has access to the manual grading page should also have access to the URL in the student context. Let me know if you want me to change this.